doc/source/templates/05_dreambooth_finetuning/dreambooth/flags.py

```diff
@@ -17,7 +17,10 @@ def train_arguments():
         type=str,
         default=None,
         required=True,
-        help="Directory where trained models are saved.",
+        help="Directory where trained models or LoRA weights are saved.",
+    )
+    parser.add_argument(
+        "--use_lora", default=False, action="store_true", help="Use LoRA."
```

Comment:
`help="Uses LoRA with default settings. The default settings are in foo.yaml"`

Reply:
going to leave this as a TODO if ok with you. Right now, probably only rank and learning rate. If we have more sophisticated setups, I will move to a dedicated yaml file.